resdx/models/nrel.py

```diff
@@ -71,6 +72,18 @@ def gross_integrated_heating_capacity(conditions, system):
     else:
       return system.gross_steady_state_heating_capacity(conditions)
 
+  @staticmethod
+  def get_cooling_cop60(conditions, system):
+    if "cooling_cop60" not in system.model_data:
+      system.model_data["cooling_cop60"] = [None]*system.number_of_speeds
+
+    if system.model_data["cooling_cop60"][conditions.compressor_speed] is not None:
+      return system.model_data["cooling_cop60"][conditions.compressor_speed]
+    else:
+      condition = system.make_condition(CoolingConditions,outdoor=PsychState(drybulb=fr_u(60.0,"°F"),wetbulb=fr_u(48.0,"°F")),compressor_speed=conditions.compressor_speed)
```

Comment:
Indoor db temperature does not change? It stays 70F?

Reply:
That's a very good question. We could make the EIR a function of indoor temperature, but I don't know how well characterized cooling performance is at these low temperatures. I think it's okay to just assume it's constant.

I did change this so that the indoor conditions are representative of heating conditions (and not cooling conditions).